### ISSUE TITLE
Support comments in Multicase TXT files

### ISSUE THREAD

**Issue by dressito** [sepinf-inc/IPED#1432]
Unlike IPED 3.XX, the multicases option in IPED 4.0.6 using a .txt file is returning an error if a line is commented with '#'.
IPED is trying to read the line as a case path and giving an 'Invalid Case Path' error.
I was using the .txt file so the user could choose which cases he would like to analyse at once, by commenting any case path he wanted to disable/ignore.
I was also using comments to 'organize' the list of case paths by company, target... and it is not possible anymore.

My txt files were something like:

```
# Comment any case you want to disable by putting a # at the beginning of the line

# Region 1
## Company A
../../path/to/case1/
../../path/to/case2/
../../path/to/case3/

## Company B
#../../path/to/case4/
#../../path/to/case5/
```

If I leave just the lines with the cases paths, everything works as expected.
Is it possible to make the multicase with txt file ignore the commented lines again?

Thanks!

**Comment by lfcnassif** [comment]
Actually comments were never supported in that file. IPED simply used to ignore invalid case paths silently, what is bad if user configures invalid ones, he would never know. So the behavior was change to abort if an invalid path is found.

But we may support comments and skip lines starting with #.

**Comment by dressito** [comment]
Hello, Luis!
Happy New Year!

I've just tested the 4.0.7 version and it is still returning an error when I have the # commented lines.
The code in Git is ok, but the source does not have the ignore String.startsWith("#") rule.
Do you have plans to implement it in the next version?

Thanks!

**Comment by dressito** [comment]
Thanks Wladimir!

**Comment by lfcnassif** [comment]
> I believe this will be included in 4.1, as usually minor releases (like 4.0.7) include mostly bug fixes.

Correct!
